<launch>
    <arg name="planning_plugin" value="sbpl_interface/SBPLPlanner"/>

    <arg name="planning_adapters" value="
            default_planner_request_adapters/FixWorkspaceBounds
            default_planner_request_adapters/FixStartStateBounds
            default_planner_request_adapters/FixStartStateCollision
            default_planner_request_adapters/FixStartStatePathConstraints"/>

            <!-- default_planner_request_adapters/AddTimeParameterization -->

    <arg name="start_state_max_bounds_error" value="0.1"/>

    <param name="planning_plugin" value="$(arg planning_plugin)"/>
    <param name="request_adapters" value="$(arg planning_adapters)"/>
    <param name="start_state_max_bounds_error" value="$(arg start_state_max_bounds_error)"/>

	<rosparam command="load" file="$(find pr2_moveit_config)/config/kinematics.yaml"/>

    <rosparam command="load" file="$(find sbpl_demos)/config/pr2/sbpl_planning2.yaml"/>

    <!-- motion primitives -->
    <group ns="graph_configs/RightArmManipLatticeConfigDefault">
		<param name="mprim_filename" value="$(find sbpl_demos)/config/pr2/pr2_arm.mprim"/>
    </group>
    <group ns="graph_configs/RightArmManipLatticeEgraphConfigDefault">
		<param name="mprim_filename" value="$(find sbpl_demos)/config/pr2/pr2_arm.mprim"/>
    </group>
	<!-- TODO -->
<!--     <group ns="graph_configs/RightArmAndTorsoManipLatticeConfigDefault"> -->
<!-- 		<param name="mprim_filename" value="$(find sbpl_demos)/config/pr2/pr2_arm_and_torso_TODO.mprim"/> -->
<!--     </group> -->

    <group ns="right_arm/planner_configs/right_arm_ARA_BFS_ML">
        <param name="plan_output_dir" value="$(env HOME)/.ros/pr2_right_arm_paths"/>
    </group>
    <group ns="graph_configs/RightArmManipLatticeEgraphConfigDefault">
        <param name="egraph_path" value="$(env HOME)/.ros/pr2_right_arm_paths"/>
    </group>
</launch>
